pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dra
+
+import (
+	"sync"
+
+	utilversion "k8s.io/apimachinery/pkg/util/version"
+)
+
+// Plugin is a description of a DRA Plugin, defined by an endpoint
+// and the highest DRA version supported.
+type Plugin struct {
+	endpoint                string
+	highestSupportedVersion *utilversion.Version
```

Comment:
Right now we don't do anything with this other than store it. I guess this is meant to future proof the implementation for if / when we need to support a new version so we can branch the implementation based on this value?

Reply:
Yes, that's how I understood it when I took this code from CSI implementation.